public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -315,7 +315,12 @@ block add-new-hero-via-detail-comp
   The user can *delete* an existing hero by clicking a delete button next to the hero's name.
   Add the following to the heroes component HTML right after the hero name in the repeated `<li>` tag:
 +makeExample('app/heroes.component.html', 'delete')
-
+:marked
+  Add the following CSS to the bottom of `heroes.component.css`.
++makeExample('toh-6/ts/app/heroes.component.css', 'delete-hero', 'app/heroes.component.css (delete)')(format=".")
+:marked
+  Add the following CSS to the bottom of `heroes.component.css`.
++makeExample('toh-6/ts/app/heroes.component.css', 'delete-hero', 'app/heroes.component.css (delete)')(format=".")
```

Comment:
Lines 318-320 are repeated in 321-323.

Also, use `makeExcerpt()` -- see Line 311 for an example.